pkg/apis/core/validation/project.go

```diff
@@ -31,6 +33,9 @@ func ValidateProject(project *core.Project) field.ErrorList {
 	allErrs := field.ErrorList{}
 
 	allErrs = append(allErrs, apivalidation.ValidateObjectMeta(&project.ObjectMeta, false, ValidateName, field.NewPath("metadata"))...)
+	if project.Name == "seed-lease" {
```

Comment:
It is gonna work. Also it wouldn't need a change in the dashboard to validate this in the frontend

Reply:
This check is also executed when existing projects are updated. Hence, if such a project already exists in a gardener landscape then it can never be updated again...

Reply:
So how can we prevent users to not specify garden--seed-lease in their `Projects`

Reply:
As suggested - use `garden--seed-lease` as namespace - all project namespaces look like `garden-<name>`, and `<name>` cannot start with `-` (I hope, please double check). WDYT?

Reply:
Yes yes I agree with that I meant the validation of the Project's namespace. Shouldn't we make it impossible to specify garden--seed-lease as a namespace when you are creating Projects via kubectl for example

Reply:
Ah, ok, I see. Yes, you can validate this in this file - just check that `.spec.namespace` is not `garden--seed-lease` (yes, this is also backwards-incompatible, but the likelihood that somebody has such a namespace that he bound to a project is ~0).